services/src/tests/integration/resourceBucket.ts

```diff
@@ -17,13 +18,29 @@ export function mockResourceBucket(initialValue: ResourceGroup, initialPolicyFil
         .reply(200, (_, body) => {
             value.current = JSON.parse(body as string) as ResourceGroup;
         })
+        .get(new RegExp(`/${bucketName!}\?.*prefix=${encodeURIComponent(policiesKeyPrefix!)}.*`))
```

Comment:
Perhaps a comment or some constants could make this more readable, because right now it's a bit hard :X